app/views/helpers/feed/update.phtml

```diff
@@ -487,11 +489,97 @@
 				<label class="group-name" for="xPathItemUid"><small><?= _t('sub.feed.kind.html_xpath.relative') ?></small><br />
 					<?= _t('sub.feed.kind.html_xpath.item_uid') ?></label>
 				<div class="group-controls">
-					<textarea class="valid-xpath w100" name="xPathItemUid" id="xPathItemUid" rows="2" cols="64" spellcheck="false"
```

Comment:
formatting changes only; unfortunately these "escaped" my checks and were committed